Framework/PythonInterface/plugins/algorithms/MuonMaxent.py

```diff
@@ -220,32 +227,69 @@ def getPhase(self,FLAGS_fixphase,POINTS_ngroups,POINTS_nhists,mylog):
             names = pt.getColumnNames()
             phaseLabel = None
             IDLabel = None
+            asymmLabel = None
             for name in names:
                 if name.lower() == "phi" or name.lower() == "phase":
                     phaseLabel = name
                 if name.lower() == "detid" or name.lower() == "spectrum number":
                     IDLabel = name
+                if name.lower() == "asymmetry" or name.lower() == "asymm" or name.lower() == "asasym":
+                    asymmLabel = name
             if phaseLabel is None:
                 raise ValueError(
                     "Phase/phi are not labelled in the phase table")
             if IDLabel is None:
                 raise ValueError(
                     "Spectrum number/DetID are not labelled in the phase table")
+            if asymmLabel is None:
+                raise ValueError(
+                    "Asymmetry is not labelled in the phase table")
 
             if(len(pt) == POINTS_ngroups):  # phase table for grouped data, or when not grouping
-                for r in pt:
-                    filePHASE[r[IDLabel] - 1] = r[phaseLabel]
+                for row in pt:
+                    filePHASE[row[IDLabel] - 1] = row[phaseLabel]
                         # sign of phase now OK for Mantid 3.12 onwards
             elif (len(pt) == POINTS_nhists):  # phase table for ungrouped data. Pick a representative detector for each group (the last one)
-                for r in pt:
-                    filePHASE[GROUPING_group[r[IDLabel] - 1]] = r[phaseLabel]
+                for row in pt:
+                    filePHASE[GROUPING_group[row[IDLabel] - 1]] = row[
+                        phaseLabel]
+            else:  # muat be some dead Detectors
+                offset = 0
+                print(POINTS_ngroups, "nnn")
```

Comment:
What's the meaning of `nnn` here?

Reply:
I think it was dubugging, I will delete it